client/dashboard/domains/domain-transfer/transfer-domain-to-other-user.tsx

```diff
@@ -188,7 +207,9 @@ export default function TransferDomainToOtherUser() {
 	};
 
 	const renderConfirmationDialog = () => {
-		const selectedUser = availableUsers.find( ( user ) => user.id.toString() === formData.user );
+		const selectedUser = availableUsers.find(
```

Comment:
It seems we can move it to the component level so we don't need to have duplicate code to get selected user.